### PR TITLE
ITREFIT-272: add Debian build support

### PR DESCRIPTION
**Pull request by RangerRick**
This PR adds support for building (and signing) Debian packages in CircleCI.

A few notes:

* `makedeb.sh` has been modified to split up building opennms/minion/centinel, like `makerpm.sh` was
* I put the Debian package runs _after_ integration tests not because they depend on it (while testing, I only depended on `build`), but because we shouldn't bother spending CPU cycles making them unless everything passes, since they're not required for anything other than being artifacts.
* This PR includes an `@Ignore` for the `WSManMonitorTest` test that's currently failing just so I could get everything to go green, but I can remove it once it passes/is approved.

### All Contributors

* [x] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [x] Have you made a comment in that issue which points back to this PR?
* [x] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [x] If this is a new or updated feature, is there documentation for the new behavior?
* [x] If this is new code, are there unit and/or integration tests?
* [x] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

Once the reviewer(s) accept the PR and the branch passes continuous integration in Bamboo, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/ITREFIT-272
* Bamboo (Continuous Integration): https://bamboo.opennms.org/

### CODE REVIEW

File: features/wsman/src/test/java/org/opennms/netmgt/poller/monitors/WSManMonitorTest.java [hunk 1]
```diff
@@ -50,6 +50,7 @@
 import com.google.common.collect.Maps;
 import com.mycila.xmltool.XMLDoc;
 
+@org.junit.Ignore
```
Comment (indigo423) [inline features/wsman/src/test/java/org/opennms/netmgt/poller/monitors/WSManMonitorTest.java]: Is this change part of the "Add Debian build support" change? Looks like this is a different topic?

File: .circleci/config.yml [hunk 2]
```diff
@@ -459,8 +473,71 @@ jobs:
...
+    resource_class: large
+    steps:
+      - attach_workspace:
+          at: ~/
+      - run:
+          name: Build Debian Packages
+          command: |
+            export NODE_OPTIONS=--max_old_space_size=1024
+            export CCI_MAXCPU=4
+            .circleci/scripts/makedeb.sh opennms
```
Comment (j-white) [inline .circleci/config.yml]: Will this end up rebuilding the `core/web-assets` project again? We had to add workaround to avoid this for the RPMs.

Comment (RangerRick) [reply .circleci/config.yml]: I think I actually fixed that with https://github.com/OpenNMS/opennms/commit/b4bdafad6e83a2b4689047361bf0d3476c6fd481 -- I'll double-check the logs though to be sure.

File: features/wsman/src/test/java/org/opennms/netmgt/poller/monitors/WSManMonitorTest.java (hunk 1 above)
Comment (RangerRick) [reply features/wsman/src/test/java/org/opennms/netmgt/poller/monitors/WSManMonitorTest.java]: See "A few notes" at the top of this PR  ;)

File: .circleci/config.yml (hunk 2 above)
Comment (RangerRick) [reply .circleci/config.yml]: Aha!  No, it still ran more than it needed to, I added the `-PskipCompile` to the Debian version of the assembly build.

File: features/wsman/src/test/java/org/opennms/netmgt/poller/monitors/WSManMonitorTest.java (hunk 1 above)
Comment (indigo423) [reply features/wsman/src/test/java/org/opennms/netmgt/poller/monitors/WSManMonitorTest.java]: > See "A few notes" at the top of this PR ;)

just jumped right into the files, I do better next time and read first - Sorry :)

**Pull request merged**